diff --git a/pacstall b/pacstall
index ac982a6..d16d6f6 100755
--- a/pacstall
+++ b/pacstall
@@ -225,12 +225,6 @@ function download() {
 	fi
 }
 
-# Error out when run as root
-if [[ $EUID -eq 0 ]]; then
-	fancy_message error "Pacstall can't be run as root. Please run as a normal user."
-	exit 1
-fi
-
 # run sudo apt update if it's been more than a week
 [ -z "$(find -H /var/lib/apt/lists -maxdepth 0 -mtime -7)" ] && sudo apt-get update -qq
 
